WooCommerce/Classes/POS/Models/ItemsStackState.swift

```diff
@@ -1,8 +1,10 @@
 import Foundation
 import Codegen
+import enum Yosemite.POSItem
 
 struct ItemsStackState {
     let root: ItemListState
+    let itemStates: [POSItem: ItemListState]
```

Comment:
Each `POSItem` will have the ability to be in a separate state. For now, it will be used for variable products to indicate that variations are loading, correct?

Reply:
Yes correct, right now just mapping from each variable product to variations (to be implemented). The pagination info might be added later when supporting pagination in the sublists. The sublist from each POS item could be any items, beyond variations.